trillian/ctfe/config_test.go

```diff
@@ -366,12 +366,14 @@ func TestToMultiLogConfig(t *testing.T) {
 					Backend: []*configpb.LogBackend{{Name: "default", BackendSpec: "spec"}},
 				},
 				LogConfigs: &configpb.LogConfigSet{
-					Config: []*configpb.LogConfig{{Prefix: "test", LogId: 1, LogBackendName: "default"}},
+					Config: []*configpb.LogConfig{
+						{LogId: 1, Prefix: "test", LogBackendName: "default"},
+					},
 				},
 			},
 		},
 		{
-			desc: "three valid log configs",
```

Comment:
The configs are not valid strictly speaking (in terms of #306) because they don't have keys. But it's okay in the scope of this test.

Reply:
Might be worth documenting that in the test.

Reply:
Done.